changelog.d/pa-1689.added

```diff
@@ -0,0 +1,4 @@
+Swift: Added parsing for ellipses in class declarations for patterns.
+
+So like:
+class $CLASS { ... }
```

Comment:
since Swift isn't even experimental yet, I haven't been writing changelog entries. it's not really in a usable state, so I don't think that the details of the work we're doing to get it to experimental matter to anyone but us.